modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -203,6 +203,31 @@ public void testAddObjectRelationship() throws Exception {
 				"able", false, ObjectRelationshipConstants.TYPE_MANY_TO_MANY,
 				null));
 
+		AssertUtils.assertFailure(
```

Comment:
Why is there an empty line between this new assert and the existing one `DuplicateObjectRelationshipException.class`?

Reply:
No reason. I removed it!